.clang-format

```diff
@@ -1,5 +1,5 @@
 #
-# The clang-format (Clang 6) style file used by deal.II.
+# The clang-format style file used by deal.II (Clang 11 instead of Clang 6)
```

Comment:
Just FYI, in preCICE we require specifically clang-format 8, so people will likely have this one installed: https://github.com/precice/precice/blob/develop/tools/formatting/format-all (I don't remember the reason)

Reply:
All in all, I think there are not that big differences between different versions. On my old machine, I used Clang 6. Updating on my recent system to Clang 11 doesn't show any differences. If the version is not suitable, we can change it later on.